.github/workflows/docker.yml

```diff
@@ -50,18 +43,11 @@ on:
            "fedora-41",
            "centos-stream-9",
            "centos-stream-9-python3.12",
-           "almalinux-8-python3.9",
            "almalinux-9-python3.11",
-           "gentoo-python3.10",
-           "gentoo-python3.11",
-           "gentoo-python3.12",
            "archlinux-latest",
            "opensuse-15.5-gcc_11-python3.11",
            "opensuse-tumbleweed-python3.10",
            "opensuse-tumbleweed",
-           "conda-forge-python3.11",
-           "ubuntu-bionic-gcc_8-i386",
-           "debian-bullseye-i386",
           ]
         # 'tox -e update_docker_platforms' updates above
       tox_packages_factors:
```

Comment:
See the comment at L52. The main list is in `tox.ini`.

Reply:
I only changed the ci flows on purpose so that people trying to fix these other systems have an easy way to do this via tox/dev container. But I can completely remove them everywhere if you don't think that anyone will actively work on them.

Reply:
> I only changed the ci flows on purpose so that people trying to fix these other systems have an easy way to do this via tox/dev container. 

Do you know that removing platforms from the CI workflow also removes the corresponding docker images?

> But I can completely remove them everywhere if you don't think that anyone will actively work on them.

You and I don't know who does and will work on what platforms. The default list of platforms tested in CI should be determined by what platforms we want to support.

Reply:
Anything that's not documented (in Sage documentation) is subject to change without notice, and our documentation does not promise anyone a million docker images ready to use. 


Thus "you and I don't know how these images are used" is not an argument to preserve them.

Reply:
I doubt if you ever read through https://doc-release--sagemath.netlify.app/html/en/developer/portability_testing.

Reply:
> I doubt whether you (and the author as well) ever read through https://doc-release--sagemath.netlify.app/html/en/developer/portability_testing.

of course I read it, I used it, etc. Please don't give such dismissive mkoeppe-style responses.

The concrete names of containers are mentioned there in *examples*. 
These are not cut in stone, they are just mentioned as examples. Just as examples might mention concrete, random, versions, random dates, etc, random outputs of computations, etc. This is not what documenting means.

Reply:
it's pretty clear that it's a collection of semi- random docker images, some of them quite old, corresponding to rather random betas, etc. 

It's too ambitious to support even half of them, too much work that noone would be doing

Reply:
There is the master list of the linux platforms we support (in the sense of @user202729 "we test it and fix it if it is broken") 

I don't care what you think should be in the list. Just update it properly by `tox -e update_docker_platforms` as the comment at Line 52 says. If not done properly, our documentation is broken. #39009 does it properly.

In the same vein, removing "minimal" and "optional" and "maximal" jobs is not a proper way to fix CI Linux. We have to support sage-the-distribution, as in the documentation.

Reply:
There is value in having this table, even if the CI currently doesn't build new images. You clearly see the latest release it was working.

Moreover, removing it from the tox.ini also removes the codespaces, which is the easiest way to actually work on these images. So if anyone want's to work on fixing the issues, then we should give people the tools to do so. This PR is not about removing support for a given system, just removing failing CI runs.